src/conduit/components/root.cljs

```diff
@@ -4,15 +4,27 @@
             [conduit.components.home :as home]
             [conduit.components.article :as article]
             [conduit.components.login :as login]
-            [conduit.components.register :as register]))
+            [conduit.components.register :as register]
+            [conduit.components.header :refer [Header]]
+            [conduit.components.footer :refer [Footer]]))
 
-(rum/defc Root < rum/reactive [r]
+(rum/defc Root < rum/reactive
+  {:did-mount
+   (fn [{[r] :rum/args :as state}]
+     (when-let [token (.getItem js/localStorage "jwt-token")]
```

Comment:
i think it's not really good, that component knows about local storage. it's better to only dispatch some action, which will describe an effect of getting local storage item and do something with it if needed. also, we already have mixin for such purposes